commit f5d5fed79aad40deefa496b52a4c77b407037999 (refs/changes/09/5409/2)
Author: Tim Foster <tim.foster@joyent.com>
Date:   2019-01-22T23:53:11+00:00 (9 months ago)
    
    TOOLS-2062 eng.git should be a git submodule of Triton/Manta repositories (fix Makefile)
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 9935014..9d0672f 100644
--- a/Makefile
+++ b/Makefile
@@ -94,12 +94,12 @@ release: all $(SMF_MANIFESTS)
 
 .PHONY: publish
 publish: release
-	@if [[ -z "$(BITS_DIR)" ]]; then \
-		@echo "error: 'BITS_DIR' must be set for 'publish' target"; \
+	@if [[ -z "$(ENGBLD_BITS_DIR)" ]]; then \
+		echo "error: 'ENGBLD_BITS_DIR' must be set for 'publish' target"; \
 		exit 1; \
 	fi
-	mkdir -p $(BITS_DIR)/registrar
-	cp $(ROOT)/$(RELEASE_TARBALL) $(BITS_DIR)/registrar/$(RELEASE_TARBALL)
+	mkdir -p $(ENGBLD_BITS_DIR)/registrar
+	cp $(ROOT)/$(RELEASE_TARBALL) $(ENGBLD_BITS_DIR)/registrar/$(RELEASE_TARBALL)
 
 
 include ./deps/eng/tools/mk/Makefile.deps
